app/src/main/java/com/mapbox/maps/testapp/examples/MapViewCustomizationActivity.kt

```diff
@@ -40,6 +41,11 @@ class MapViewCustomizationActivity : AppCompatActivity() {
       )
       .build()
 
+    // plugins configuration
+    val plugins = listOf(
+      PLUGIN_LOGO_CLASS_NAME
```

Comment:
Attribution is also mandatory for majority of cases.

Reply:
for you I added it! but it's really not a requirement for the testapp :upside_down_face: